bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,259 @@ class CoffeeLintBear:
+            Prevents defining duplicate keys in object literals and classes.
+        :param: enforce_parentheses_on_empty_constructor:
+            Requires constructors with no parameters to include parentheses.
+        :param: line_endings:
+            Ensures that the file uses only ``windows`` or ``unix`` line
+            endings.
+        :param: line_endings_option:
+            The option to ``line_endings``, its value is either ``unix`` or
+            ``windows``.
+        :param: no_this:
+            prohibits ``this``. Use ``@`` instead.
+        :param: no_plusplus:
+            Forbids the increment and decrement arithmetic operators.
+        :param: no_empty_parameter_list:
+            Prohibits empty parameter lists in function definitions.
+        :param: no_empty_functions:
+            Disallows declaring empty functions.
+        """
+        coffee_configs = {"max_line_length":
+                          {"value": max_line_length,
+                           "level": "error",
+                           "limitComments":
+                               apply_max_line_length_on_comments}}
+        if space_before_and_after_arrow:
+            coffee_configs["arrow_spacing"] = {"level": "ignore"}
+        if braces_spacing:
+            coffee_configs["braces_spacing"] = (
+                {"level": "ignore",
+                 "spaces": braces_spacing_width,
+                 "empty_object_spaces": spacing_in_empty_braces})
+        if camelcase_classes_name:
+            coffee_configs["camel_case_classes"] = {"level": "error"}
+        if spaces_before_and_after_colon:
+            coffee_configs["colon_assignment_spacing"] = (
+                {"level": "ignore",
+                 "spacing": {"left": spaces_before_colon,
+                             "right": spaces_after_colon}})
+        if end_file_with_a_single_newline:
+            coffee_configs["eol_last"] = {"level": "ignore"}
+        if newlines_after_classes:
+            coffee_configs["newlines_after_classes"] = (
+                {"value": number_of_lines_after_classes,
+                 "level": "ignore"})
+        if no_backticks:
+            coffee_configs["no_backticks"] = {"level": "error"}
+        if no_implicit_braces:
+            coffee_configs["no_implicit_braces"] = (
+                {"level": "ignore", "strict": True})
+        if no_function_call_without_parentheses:
+            coffee_configs["no_implicit_parens"] = (
+                {"strict": True, "level": "ignore"})
+        if no_interpolation_in_single_quotes:
+            coffee_configs["no_interpolation_in_single_quotes"] = (
+                {"level": "ignore"})
+        if no_stand_alone_at:
+            coffee_configs["no_stand_alone_at"] = {"level": "ignore"}
+        if no_tabs:
+            coffee_configs["no_tabs"] = {"level": "error"}
+            coffee_configs["indentation"] = (
+                          {"value": 2, "level": "error"})
```

Comment:
what I mean is that `4` is a common indentation, but if it's common to use 2 for coffee-script files, alright